README.md

```diff
@@ -76,6 +76,163 @@ For more info about the API, see the [API reference](https://kontent.ai/learn/re
 
 You can find the Delivery and other SDKs at <https://github.com/Kentico>.
 
+## About
+
+This section describes the application.

```

Comment:
We should either summarize the content of the section in this part, or remove this sentence - it is ambignous